### PR TITLE
PMM-3763 Adding Jobs for PMM Documentation

### PR DESCRIPTION
**Pull request by puneet0191**

### CODE REVIEW

**Comment by hors** [comment]
please see comment:  https://jira.percona.com/browse/PMM-3766?focusedCommentId=237322&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-237322

File: pmm/pmm-1-0-documentation-pdf.groovy [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+            steps {
+                withCredentials([sshUserPrivateKey(credentialsId: 'aws-jenkins', keyFileVariable: 'KEY_PATH', passphraseVariable: '', usernameVariable: 'USER')]) {
+                    sh '''
+                        sudo yum -y install docker
+                        sudo usermod -aG docker ec2-user
+                        sudo service docker start
+                        echo BRANCH = \${BRANCH_NAME}
+                        echo "Building in: " `pwd`
+                        cd doc
+                        sudo docker build . --tag pmm_1_0_doc_docker_image
```
Comment (hors) [inline pmm/pmm-1-0-documentation-pdf.groovy]: Why do you build docker image every time when you run the job? Maybe it is better to build it once and push it to docker hub (Percona Lab) .

Comment (puneet0191) [reply pmm/pmm-1-0-documentation-pdf.groovy]: Tried, Cna't push to hub it seems, check this https://pmm.cd.percona.com/view/all/job/doc-test/43/console 
`denied: requested access to the resource is denied`

File: pmm/pmm-2-0-documentation-pdf.groovy [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    agent {
+        label 'micro-amazon'
+    }
+    parameters {
+        string(
+            defaultValue: 'git@github.com:percona/pmm-doc',
+            description: 'Repository name for build',
+            name: 'REPO_NAME')
+        string(
+            defaultValue: 'master',
```
Comment (hors) [inline pmm/pmm-2-0-documentation-pdf.groovy]: PMM-2.0 ?

File: pmm/pmm-2-0-documentation.groovy [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+    agent {
+        label 'micro-amazon'
+    }
+    parameters {
+        string(
+            defaultValue: 'git@github.com:percona/pmm-doc',
+            description: 'Repository name for build',
+            name: 'REPO_NAME')
+        string(
+            defaultValue: 'master',
```
Comment (hors) [inline pmm/pmm-2-0-documentation.groovy]: PMM-2.0 ?

Comment (puneet0191) [reply pmm/pmm-2-0-documentation.groovy]: yes, Repo's are different for docs of both version, pmm1.0 and pmm2.0

File: pmm/pmm-2-0-documentation-pdf.groovy [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+            steps {
+                withCredentials([sshUserPrivateKey(credentialsId: 'aws-jenkins', keyFileVariable: 'KEY_PATH', passphraseVariable: '', usernameVariable: 'USER')]) {
+                    sh '''
+                        sudo yum -y install docker
+                        sudo usermod -aG docker ec2-user
+                        sudo service docker start
+                        echo BRANCH = \${BRANCH_NAME}
+                        echo "Building in: " `pwd`
+                        cd doc
+                        sudo docker pull perconalab/pmm-doc
```
Comment (hors) [inline pmm/pmm-2-0-documentation-pdf.groovy]: Maybe we can use sg docker -c "...."  to avoid using 'sudo' command...

Comment (puneet0191) [reply pmm/pmm-2-0-documentation-pdf.groovy]: done please check again

**Pull request merged**